metrics/metrics-api/src/main/java/org/apache/james/metrics/api/MeterMetric.java

```diff
@@ -0,0 +1,26 @@
+/****************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ *                                                              *
+ *   http://www.apache.org/licenses/LICENSE-2.0                 *
+ *                                                              *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ ****************************************************************/
+
+package org.apache.james.metrics.api;
+
+public interface MeterMetric {
```

Comment:
We discussed together that we should not add another "mtric" object but rather find a generic formulation to allow "moving average implementation" with a naive "getCount" call.

Reply:
I misunderstood as well then... Why should add our own complex implementation instead of reusing `Meter` from DropWizard that does exactly what we want?

Reply:
(because we need to find how to expose this functionality but without enforcing dropwizard dependency everywhere, we need an indirection interface, and this is what metrics projects are for)